README.md

```diff
@@ -1,6 +1,26 @@
 # colcon-in-container
 
-colcon verb extension to build and test inside a container and transfer the results back to the host
+colcon verb extension to build and test inside a fresh and isolated ROS environment and transfer the results back to the host.
+
+With this extension, developers can build ROS packages for any ROS 2 distributions directly from colcon independently of the host. With it, one can validate builds and tests making sure all the dependencies are properly listed in their `package.xml` on any ROS 2 distribution. Validating packages and workspace in an isolated and ephemeral environment is key for distributing and packaging software.
+
+## Quickstart
+
+- Install the tool:
+```
+pip3 install -U git+https://github.com/canonical/colcon-in-container
+```
+- Install and initialize LXD:
+```
+sudo snap install lxd
+lxd init --auto
+```
+- At the root of the ROS 2 workspace
```

Comment:
Why necessarily at the ws' root? Can't I build a single package directly from that package directory?

Reply:
Indeed, it works. I will change that.